src/main/java/frc/robot/commands/autoCommands/waitCommand.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands.autoCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class waitCommand extends Command {
+
+  private int delay;
+  private int curTimeWaited = 0;
+
+  public waitCommand(int delay) {
+    this.delay = delay;
+  }
+
+  @Override
+  public void initialize() {
+    this.curTimeWaited = 0;
+  }
+
+  @Override
+  public void execute() {
+    this.curTimeWaited++;
```

Comment:
This is a big gotcha. Never assume that a command will run consistently in a pseudo real time system. In this case, 
you don't actually know if the command will run every 20 ms. It _should_, but say you get a loop overrun. What happens next is the next time it runs is actually closer to 40 ms away from the previous time and that delays your command.

The [Timer class](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/Timer.html) is used internally by the WaitCommand and actually handles those sorts of issues a little better.